benchmarks/pool_stress_test.cpp

```diff
@@ -72,7 +72,13 @@ template <class T>
 void do_test(std::string pool_name, std::map<const std::string, const std::vector<std::size_t>&> indexing_pairs)
 {
   auto& rm {umpire::ResourceManager::getInstance()};
+
+#if defined (UMPIRE_ENABLE_DEVICE)
```

Comment:
I think always using HOST would be fine here.

Reply:
Oh wow, even easier haha - okay!